lte/gateway/release/update_deb_meta_info.sh.template

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
+
+################################################################################
+# Copyright 2022 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+set -euo pipefail
+
+###############################################################################
+# FUNCTION DECLARATIONS
+###############################################################################
+
+update_version() {
+    local INPUT_FILE_NAME=$1
+
+    rm -rf "${WORK_FOLDER}"
+    mkdir "${WORK_FOLDER}"
+    cd "${WORK_FOLDER}"
+
+    echo "Updating meta data in ${INPUT_FILE_NAME} ..."
```

Comment:
Yes, it is - so that follow up output is associated to this line. Sample output:
```
Updating meta data in magma_1.9.0-VERSION-SUFFIX_amd64.deb ...
  Updating version "1.9.0-VERSION-SUFFIX" to "1.9.0-1667398648-cd2b07e9"
  Updating description to development build "false"
  Updating "COMMIT_HASH" in magma to "magma@1.9.0.11665-cd2b07e9"
  Renaming "magma_1.9.0-VERSION-SUFFIX_amd64.deb" to "magma_1.9.0-1667398648-cd2b07e9_amd64.deb"
  File written to /home/vscode/magma/magma_1.9.0-1667398648-cd2b07e9_amd64.deb
Updating meta data in magma-sctpd_1.9.0-VERSION-SUFFIX_amd64.deb ...
  Updating version "1.9.0-VERSION-SUFFIX" to "1.9.0-1667398648-cd2b07e9"
  Updating description to development build "false"
  Renaming "magma-sctpd_1.9.0-VERSION-SUFFIX_amd64.deb" to "magma-sctpd_1.9.0-1667398648-cd2b07e9_amd64.deb"
```